pkg/functions/client.go

```diff
@@ -139,13 +140,14 @@ type Runner interface {
 // Remover of deployed services.
 type Remover interface {
 	// Remove the function from remote.
-	Remove(ctx context.Context, name string, namespace string) error
+	Remove(ctx context.Context, name string, namespace string) (bool, error)
 }
 
 // Lister of deployed functions.
 type Lister interface {
 	// List the functions currently deployed.
-	List(ctx context.Context, namespace string) ([]ListItem, error)
+	// the 2nd argument (bool) indicates it the lister had to handle any function at all
```

Comment:
I am forgetting why the `bool` was necessary?  What is the circumstance in which `ok==true` but `len(items)==0`?